.travis.yml

```diff
@@ -35,9 +38,12 @@ before_script:
 script:
   - bash tests/bin/phpunit.sh
   - bash tests/bin/phpcs.sh
+  - bash tests/e2e-tests/scripts/run-e2e.sh  
 
 after_script:
   - bash tests/bin/travis.sh after
+  - export NODE_CONFIG_DIR="./tests/e2e-tests/config"
```

Comment:
What do you think if we move this line and the line below to `tests/bin/travis.sh`? It seems to me that it would be better to keep all the code that is executed "after_script" in a single place.